server/views/applications/partials/_timeline.njk

```diff
@@ -42,7 +42,17 @@
           <div class="moj-timeline__description">
             <p>{{event.content}}</p>
           </div>
-          {%endif%}
+       {%endif%}
+       {% if event.associatedUrls %}
+         {% for associatedUrl in event.associatedUrls %}
+            <div class="moj-timeline__description">
+              <ul class="govuk-list govuk-list--bullet">
+                <li><a class="govuk-link" href="{{associatedUrl.url}}">{{associatedUrl.type}}</a></li>
```

Comment:
Can we make the link text a bit friendlier here? So it says `View application` instead of just `application`?